apple/RNCScriptMessageManager.m

```diff
@@ -34,19 +34,19 @@ @interface RNCScriptMessageManager() {
 @implementation RNCScriptMessageManager
 
 + (id) sharedManager {
-    static RNCScriptMessageManager *_sharedManager = nil;
-    @synchronized(self) {
-      if(_sharedManager== nil) {
-        _sharedManager = [[super alloc] init];
-      }
-      return _sharedManager;
+  static RNCScriptMessageManager *_sharedManager = nil;
+  @synchronized(self) {
+    if(_sharedManager== nil) {
```

Comment:
nit: `_sharedManager == nil`

Reply:
good catch! I'll update this

Reply:
https://github.com/discord/react-native-webview/pull/25/commits/d7a448927a365fc799c790a329fddb90034776e1